docs/pages/visualization.md

```diff
@@ -0,0 +1,34 @@
+@page visualization Visualization Guide
+
+[TOC]
+
+# About
+
+svZeroDVisualization is a web application designed for visualizing 0D simulation results and the 0D network. It allows users to interactively explore and analyze their simulation data through an intuitive interface. 
+This application is available in the  `applications` folder.
+
+
+# Architecture
+svZeroDVisualization is built using a robust architecture that includes:
+- Frontend: Utilizes HTML, CSS, Dash, and Plotly for creating a dynamic and interactive user interface. This setup allows for effective visualization and interaction with the 0D network and simulation results.
+- Backend: Powered by a Flask application that handles the server-side logic. It leverages NetworkX for managing and visualizing the network graph and a Python version of svZeroDSolver to draw connections.
```

Comment:
I would change "a Python version of svZeroDSolver" to "a Python code". We don't want users to think we have multiple versions of svZeroDSolver. Also, the Python code is just meant to parse the json file and figure out the connections, right? It does not draw the connections?

Reply:
Gotcha makes sense! True - it's not drawing connections so I've changed it to say "and a Python code to determine network connections."